ng2c3.component.ts.ts

```diff
@@ -0,0 +1,80 @@
+import {Component, Input, SimpleChange, ElementRef, ViewEncapsulation} from '@angular/core';
+import * as c3 from 'c3';
+
+
+/**
+ * Ng2-C3 angular2 directive for C3 charts
+ *
+ * Usage:
+ * eg.) <ng2-c3 bind-data="dataFromTheComponentOrWherever"></ng2-c3>
+ *
+ * 1. MultiLine chart takes the parameters
+ *      a. data of c3 configuration
+ */
+  //TODO: Handle more generic items for various other types of charts
+@Component({
+  moduleId: module.id,
+  selector: 'ng2-c3',
+  template: ``,
+  encapsulation: ViewEncapsulation.None
+})
+export class Ng2C3 {
+
+  @Input()
+  private data : any; // Configuration for series to be used for generating C3 has to be here
+  private element : HTMLElement; // Element to which the chart has to be attached to
+
+  constructor(elementReference : ElementRef) {
+    this.element = elementReference.nativeElement;
+  }
+
+  private __render( inputData : Map<string, any>) : void {
```

Comment:
Can you modify the format of inputData to any? Same has been used at top for data